StripePayments/StripePayments/Source/PaymentHandler/STPPaymentHandler.swift

```diff
@@ -151,7 +151,7 @@ public class STPPaymentHandler: NSObject {
 
     private var formSpecPaymentHandler: FormSpecPaymentHandler?
 
-    internal var _redirectShim: ((URL, URL?, Bool) -> Void)?
+    public var _redirectShim: ((URL, URL?, Bool) -> Void)?
```

Comment:
Are we sure we want to change this to public? This was largely here for testing.

Reply:
Definitely don't want to make this public, thanks for catching!